CHANGELOG.md

```diff
@@ -115,37 +159,93 @@ Your project may not depend on some of the packages listed in this table.
+[3588]: https://github.com/magento/pwa-studio/pull/3588
+[3580]: https://github.com/magento/pwa-studio/pull/3580
+[3594]: https://github.com/magento/pwa-studio/pull/3594
+[9]: https://github.com/magento-commerce/venia-sample-data-modules/pull/9
+[3486]: https://github.com/magento/pwa-studio/pull/3486
+[3]: https://github.com/magento-commerce/magento2-pwa/pull/3
+[9]: https://github.com/magento-commerce/magento2-pwa/pull/9
+[7]: https://github.com/magento-commerce/magento2-pwa/pull/7
+[8]: https://github.com/magento-commerce/magento2-pwa/pull/8
+[3576]: https://github.com/magento/pwa-studio/pull/3576
+[3616]: https://github.com/magento/pwa-studio/pull/3616
+[3593]: https://github.com/magento/pwa-studio/pull/3593
+[3598]: https://github.com/magento/pwa-studio/pull/3598
+[3577]: https://github.com/magento/pwa-studio/pull/3577
+[3589]: https://github.com/magento/pwa-studio/pull/3589
+[3601]: https://github.com/magento/pwa-studio/pull/3601
+[3597]: https://github.com/magento/pwa-studio/pull/3597
+[3586]: https://github.com/magento/pwa-studio/pull/3586
+[3526]: https://github.com/magento/pwa-studio/pull/3526
+[3533]: https://github.com/magento/pwa-studio/pull/3533
+[3525]: https://github.com/magento/pwa-studio/pull/3525
+[36]: https://github.com/magento-commerce/pwa-tests/pull/36
+[3518]: https://github.com/magento/pwa-studio/pull/3518
+[37]: https://github.com/magento-commerce/pwa-tests/pull/37
+[3500]: https://github.com/magento/pwa-studio/pull/3500
+[27]: https://github.com/magento-commerce/pwa-tests/pull/27
+[3491]: https://github.com/magento/pwa-studio/pull/3491
+[3483]: https://github.com/magento/pwa-studio/pull/3483
+[11]: https://github.com/magento-commerce/venia-sample-data-modules/pull/11
+[3465]: https://github.com/magento/pwa-studio/pull/3465
+[1]: https://github.com/magento-untitled-geese/venia-sample-data-modules/pull/1
+[3521]: https://github.com/magento/pwa-studio/pull/3521
+[3544]: https://github.com/magento/pwa-studio/pull/3544
+[13]: https://github.com/magento-commerce/venia-sample-data-modules/pull/13
+[3538]: https://github.com/magento/pwa-studio/pull/3538
+[3529]: https://github.com/magento/pwa-studio/pull/3529
+[11]: https://github.com/magento-commerce/magento2-upward-connector/pull/11
+[3559]: https://github.com/magento/pwa-studio/pull/3559
+[39]: https://github.com/magento-commerce/pwa-tests/pull/39
+[3564]: https://github.com/magento/pwa-studio/pull/3564
+[3555]: https://github.com/magento/pwa-studio/pull/3555
+[3552]: https://github.com/magento/pwa-studio/pull/3552
+[101]: https://github.com/magento-commerce/pwa-studio-cicd/pull/101
+[3561]: https://github.com/magento/pwa-studio/pull/3561
+[15]: https://github.com/magento-commerce/magento2-pwa/pull/15
+[3654]: https://github.com/magento/pwa-studio/pull/3654
+[3624]: https://github.com/magento/pwa-studio/pull/3624
+[14]: https://github.com/magento-commerce/magento2-pwa/pull/14
+[3619]: https://github.com/magento/pwa-studio/pull/3619
+[]: <>
```

Comment:
```suggestion

```

Reply:
I think this might have messed up the links somehow, they're getting rendered at the bottom of the page:

![Screen Shot 2022-01-28 at 10 50 28 AM](https://user-images.githubusercontent.com/13182778/151588093-7943256e-84c3-4422-a88a-7be460c0d903.png)